app/assets/javascripts/views/story_view.js

```diff
@@ -382,6 +383,12 @@ module.exports = FormView.extend({
     storyElement.effect("highlight", {color: 'lightgreen'}, 1500);
   },
 
+  highlightCreatedStories: function () {
+    let storyID = `#story-${ this.model.get('id') }`;
+    let storyElement = $(storyID);
+    storyElement.effect("highlight", {color: 'lightgreen'}, 1500);
```

Comment:
This could be reused, it's duplicated here https://github.com/Codeminer42/cm42-central/pull/312/files#diff-5d8aecce3a76454de0b472c96fa1b6a8R383, what do you think about creating a `highlightStory()` method?